### DISCUSSION TITLE
Feedback: modulePrefixes in tsconfig.json

### DISCUSSION THREAD

**Discussion by johnf** [microsoft/rnx-kit#1971] [Ideas]
You are likely here because ```@rnx-kit/cli``` has thrown the following error:

```
Error: When using Typescript >=4.7, @rnx-kit/cli sets modulePrefixes. It does not currently support setting of modulePrefixes unless it exactly matches [".android", ".native", ""] or [".ios", ".native", ""]. We would like to understand any use cases where you are setting modulePrefixes to something else. Please comment at TBA
```
Please provide feedback in this discussion so that we can cater for more use cases.

**Comment by tido64** [comment]
Hi @johnf, sorry for the super late reply. Fwiw, this code has been removed now: https://github.com/microsoft/rnx-kit/pull/2544/files#diff-cfcc630699909cb3059f079b69591695a6384003577d1ae9d696010f5da644ed. We previously thought that ensuring that declared platforms are consistent in transitive dependencies made sense, but this doesn't reflect how `moduleSuffixes` can be used.
